apps/evm/src/pages/Swap/__tests__/index.spec.tsx

```diff
@@ -672,10 +676,39 @@ describe('Swap', () => {
     fireEvent.click(submitButton!);
 
     // Check swap was executed
-    await waitFor(() => expect(swapTokens).toHaveBeenCalledTimes(1));
-    expect(swapTokens).toHaveBeenCalledWith({
-      swap: fakeExactAmountInSwap,
-    });
+    await waitFor(() => expect(mockSwapTokens).toHaveBeenCalledTimes(1));
+    expect(mockSwapTokens.mock.calls[0]).toMatchInlineSnapshot(`
```

Comment:
nit: wouldn't it be better to just snapshot these instead of having them inline?

Reply:
I pointed out some other files that have them, other than that PR looks good!